src/cadnano_oxDNA.py

```diff
@@ -653,18 +653,19 @@ def parse_cadnano(path):
     def print_usage():
         print("USAGE:", file=sys.stderr)
         print("\t%s cadnano_file lattice_type" % sys.argv[0], file=sys.stderr)
-        print("\t[-q\--sequence FILE] [-b\--box VALUE] [-e\--seed VALUE] [-p\--print-virt2nuc]", file=sys.stderr) 
+        print("\t[-q\--sequence FILE] [-b\--box VALUE] [-e\--seed VALUE] [-p\--print-virt2nuc] [-v\--print-oxview]", file=sys.stderr) 
```

Comment:
I'd rather not use `-v`, which is usually reserved to get the script's version (although we haven't implemented anything of the sort here). Can we use something else, for instance `-o`?

Reply:
Good point, I've changed it now. I think `-o` tend to be used for specifying output files, but I think it should be alright anyway